File: hackerrank/easy/0058-camelcase/js/solution.mjs
// solution.mjs

export function camelcase(s) {
  if (!s || s.length === 0) return 0;

  let count = 1;
  for (let i = 0; i < s.length; i++) {
    const ch = s[i];
    if (ch >= "A" && ch <= "Z") {
      count++;
    }
  }
  return count;
}

// --- demo run ---
if (import.meta.url === `file://${process.argv[1]}`) {
  const sample = "saveChangesInTheEditor"; // Expected: 5
  console.log(camelcase(sample));
}

File: hackerrank/easy/0058-camelcase/python/solution.py
## solution.py

def camelcase(s: str) -> int:
    if not s:
        return 0

    count = 1
    for ch in s:
        if ch.isupper():
            count += 1
    return count


# --- demo run ---
if __name__ == "__main__":
    sample = "saveChangesInTheEditor"  # Expected: 5
    print(camelcase(sample))
